engine/apps/alerts/paging.py

```diff
@@ -163,15 +163,16 @@ def unpage_user(alert_group: AlertGroup, user: User, from_user: User) -> None:
             ).select_for_update()[0]
```

Comment:
I'm not entirely sure the purpose of `UserHasNotification` but when testing locally there were several cases where `POST /unpage_user` wouldn't actually "unpage" the user because the log record was never created due to an `IndexError` being thrown.

This change ensures that this log record is always created.